### DISCUSSION TITLE
[QUESTION] OS/2 Peer Networking in 86Box

### DISCUSSION THREAD

**Discussion by ppgrainbow** [86Box/86Box#1322] [Q&A]
First of all, I would like to thank TC1995 for fixing the S3 hardware cursor bug on OS/2 Warp 3 and I really appreciate it!

Now, it's time for a different discussion. I have been trying to get OS/2 Peer to work under OS/2 Warp 3 and efforts to do so haven't been successful.

When I try to start up OS/2 Warp 3, I got this error message:
![20210324-042054-814](https://user-images.githubusercontent.com/32658796/112270789-5c9fd280-8c37-11eb-8b6d-b1b92c985bc1.png)

Additional errors occur when attempting to start OS/2 Peer stating that it can't be started properly. I'm currently using SLiRP and while you can't access network shares from a OS/2 Warp 3 VM on 86Box, but I don't know if I can access network shares from other VMs that I have running in the background such as VMware and VirtualBox.

Here are the contents of my CONFIG.SYS,, PROTOCOL.INI, IBMLAN.INI and MPTCONFG.INI files:

**1.** https://pastebin.com/Zq0tNSUw
**2.** https://pastebin.com/1VQLVLMb
**3.** https://pastebin.com/Fehq4NCq
**4.** https://pastebin.com/HMKLavwp

So, what should I do to get OS/2 Peer Networking working on 86Box? If for some reason, there is no way to get Peer networking working, then I might as well uninstall it.

Thank you for your time. :)

**Comment by ppgrainbow** [comment]
And here is another error that I got when attempting to start PEER:

![20210325-210311-642](https://user-images.githubusercontent.com/32658796/112543415-f324de80-8d72-11eb-9017-1f9d8b97f03c.png)

Any ideas?

**Comment by rakslice** [comment]
In case it's still relevant: Is the path there correct? Is the file actually on `D:\` at the location specified in the error message?
